src/image/Wcs.cc

```diff
@@ -1082,7 +1082,8 @@ void Wcs::write(OutputArchiveHandle & handle) const {
     WcsPersistenceHelper const & keys = WcsPersistenceHelper::get();
     afw::table::BaseCatalog catalog = handle.makeCatalog(keys.schema);
     PTR(afw::table::BaseRecord) record = catalog.addNew();
-    record->set(keys.crval, getSkyOrigin()->getPosition(afw::geom::degrees));
+    record->set(keys.crval.getX(), _wcsInfo[0].crval[0]);
```

Comment:
It's sad that we can't use our Sky methods, but I guess it makes sense to do it this way in the internals.